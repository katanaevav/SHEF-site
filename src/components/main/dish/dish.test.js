import React from "react";
import renderer from "react-test-renderer";
import Dish from "./dish.jsx";

it(`Render Dish`, () => {
  const tree = renderer.create(
      <Dish
        dishId = {1}
        dishNmae = {`Анатомия вкуса`}
        dishDescription = {`Описание блюда Описание блюда
        Описание блюда Описание блюда
        Описание блюда Описание блюда
        Описание блюда Описание блюда`}
        dishPrice = {`250 р.`}
        dishWeight = {`250 гр.`}
        dishTag = {`NEW`}
        dishImage = {`./img/dish.png`}
        dishImage2x = {`./img/dish@2x.png`}
        MenuCategory = {1}
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
