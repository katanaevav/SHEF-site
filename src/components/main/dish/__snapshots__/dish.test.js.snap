// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Dish 1`] = `
<li
  className="menu-group-navigator__item dish"
>
  <div
    className="dish-wrapper dish-wrapper--head"
  >
    <p
      className="dish-weight"
    >
      250 гр.
    </p>
    <p
      className="dish-tag"
    >
      NEW
    </p>
  </div>
  <img
    alt="Анатомия вкуса"
    className="dish__image"
    height="166"
    src="./img/dish.png"
    srcSet="./img/dish@2x.png 2x"
    width="166"
  />
  <div
    className="dish__counter select-count "
  >
    <button
      className="select-count__button select-count__button--dec"
      onClick={[Function]}
    >
      -
    </button>
    <input
      className="select-count__input"
      defaultValue={1}
      max={99}
      min={1}
      onChange={[Function]}
      type="number"
    />
    <button
      className="select-count__button select-count__button--inc"
      onClick={[Function]}
    >
      +
    </button>
  </div>
  <h3
    className="dish__header"
  >
    Анатомия вкуса
  </h3>
  <p
    className="dish__description"
  >
    Описание блюда Описание блюда
                    Описание блюда Описание блюда
                    Описание блюда Описание блюда
                    Описание блюда Описание блюда
  </p>
  <div
    className="dish-wrapper dish-wrapper--footer"
  >
    <p
      className="dish__price"
    >
      250 р.
    </p>
    <button
      className="dish__to-cart"
    >
      В корзину
    </button>
  </div>
</li>
`;
